core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -149,6 +152,14 @@
   }
 
   /* All classes below should start with .conversation-skin */
+  .conversation-skin-correctness-footer-container {
+    padding: 0px;
```

Comment:
Use 0 instead of 0px. Also, alphabetize (bottom, then padding, etc.).

Reply:
Done!